plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/ProgramConverter.kt

```diff
@@ -30,6 +28,7 @@ import org.jetbrains.kotlin.formver.viper.ast.*
  */
 class ProgramConverter(val session: FirSession, override val config: PluginConfiguration) : ProgramConversionContext {
     private val methods: MutableMap<MangledName, MethodEmbedding> = mutableMapOf()
+    private val methodsToVerify: MutableList<FirFunctionSymbol<*>> = mutableListOf()
```

Comment:
I think the choice which methods to verify and which not to is the made in the `Verifier`. This is more like `methodsToConvert` right? Also an explaining comment distinguishing `methods` and `methodsToVerify` could be helpful.

Reply:
Hm, that's an interesting distinction.  I suppose my thinking was that we are to some degree converting many more methods, but just not their bodies.  I'll think about a better name.